### DISCUSSION TITLE
support ppstructure in chinese

### DISCUSSION THREAD

**Discussion by ZTurboX** [PaddlePaddle/PaddleOCR#6576] [General]
hope to support ppstructure in chinese

**Comment by Evezerest** [comment]
hi, PP-Structure does support Chinese table recognition. More complex table structure recognition will be available in the next version

**Comment by WenmuZhou** [comment]
we have support it now, you can see it in https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/ppstructure/README.md

**Reply by TapendraBaduwal** [reply]
@Evezerest 
@WenmuZhou 
@ZTurboX 

Dear all
How to recognition multiple languages text in the same cell coordinates of table? It is possible to use language classification model in this case?
